src/App.jsx

```diff
@@ -1,5 +1,9 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
+import { GoodList } from './compotents/GoodList/GoodList';
+import { Reset } from './compotents/Reset/Reset';
```

Comment:
The import paths for `GoodList` and `Reset` components contain a typo in the folder name `compotents`. It should likely be `components`.